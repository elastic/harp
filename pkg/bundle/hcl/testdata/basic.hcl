package "app/credentials/database" {
    description = ""

    secrets = {
        "USER" = "{{ strongPassword }}"
    }
}
