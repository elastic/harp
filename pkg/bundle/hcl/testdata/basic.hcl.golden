(*hcl.Config)({
 Annotations: (map[string]string) <nil>,
 Labels: (map[string]string) <nil>,
 Packages: ([]hcl.Package) (len=1 cap=1) {
  (hcl.Package) {
   Path: (string) (len=24) "app/credentials/database",
   Description: (string) "",
   Annotations: (map[string]string) <nil>,
   Labels: (map[string]string) <nil>,
   Secrets: (map[string]string) (len=1) {
    (string) (len=4) "USER": (string) (len=20) "{{ strongPassword }}"
   }
  }
 }
})
